### ISSUE TITLE
The base link does not work anymore

### ISSUE THREAD

**Issue by armathur** [mdancho84/analysis_Russell2000#1]
The base link 
http://www.marketvolume.com/indexes_exchanges/r2000_components.asp?s=RUT&row=
does not provide pages

**Comment by mdancho84** [comment]
Yes, this is an issue with the Market Volume website in which the stock components are no longer publicly available. We are working on a solution. In the meantime, you can use `tidyquant::tq_index("RUSSELL2000", use_fallback = TRUE)`.
